packages/fx-core/src/common/featureFlags.ts

```diff
@@ -62,6 +62,10 @@ export function isCopilotAuthEnabled(): boolean {
   return featureFlagManager.getBooleanValue(FeatureFlags.CopilotAuth);
 }
 
+export function isSMEOAuthEnabled(): boolean {
```

Comment:
Suggest to use unified global feature flag manager directly: featureFlagManager.getBooleanValue(FeatureFlags.SMEOAuth)

Reply:
As discussed, we plan to update it in another PR